docs/start/envwin.md

```diff
@@ -1,10 +1,6 @@
 # ![win](../res/win_med.png) Windows System Pre-requisites
 
-## Windows 8 and Beyond (64-bit)
-
-No known system pre-requisistes are known at this time.
-
-## Windows 7 and Windows Server 2012 R2 (64-bit)
+## Windows 7 and above including Windows Server 2012 R2 (64-bit)
```

Comment:
We had confirmed this in the past.  Make the PR reflect that and if something changes, we will update it.